### DISCUSSION TITLE
Windows 10 or 11 - New Xournal++ File from the Right Click Context Menu

### DISCUSSION THREAD

**Discussion by LHSequalsRHS** [xournalpp/xournalpp#5822] [General]
Hey,

I tinkered with the regedit to add a new file from the right click context menu in Windows. However, when opening the file I always receive the error message - "Error opening file ""/home/user/file.xopp"" Document is not complete (maybe the end is cut off?)".

Anyone have a better solution for this? 

Thanks

**Comment by PillowTrauma** [comment]
Microsoft made a handy program called PowerToys. One of its features is New+, which sets up a Templates folder in which you can put any combination of file types and folder structures, and they will appear in the right click menu under New+.
